syseng/softwaresetup.md

```diff
@@ -88,7 +88,7 @@ If the tool runs without error, you're ready to get started!
 
 ## First 5 Things to Do
 
-Now that your Grayskull™ e75/e150 or Wormhole™ n150/n300 AI graph processor is installed and up and running, there are two SDKs you can work with to familiarize yourself with the hardware and associated utilities and software. Each SDK lists five tasks to start with.
+Now that your Tenstorrent AI graph processor is installed and up and running, there are two SDKs you can work with to familiarize yourself with the hardware and associated utilities and software. Each SDK lists five tasks to start with.
 
 - [First 5 Things](https://github.com/tenstorrent/tt-buda-demos?tab=readme-ov-file#first-5-things-to-do) for **TT-Buda**, our high level SDK
-- [First 5 Things](https://tenstorrent.github.io/tt-metalium/latest/get_started/get_started.html) for **TT-Metalium**, our open source, low level SDK
\ No newline at end of file
+- [First 5 Things](/tt-metalium/latest/get_started/get_started.html) for **TT-Metalium**, our open source, low level SDK
```

Comment:
This part should work but unfortunately does not and is interpreted as an anchor tag.
<img width="772" alt="Screenshot 2024-08-12 at 5 49 05 PM" src="https://github.com/user-attachments/assets/59f24846-6e5b-4922-bd94-f853ef5e4445">

Reply:
that's thoroughly obnoxious, let me go poke at the markdown engine as that's just weird

Reply:
This seems to be the issue in the .md files but not .rst.

Reply:
Ok so I know what's going on but it's somewhat broken given how sphinx wants to work.  Basically sphinx is overriding the link to try and make it a sphinx reference, ok good so far.  Try to flip this to a reference and yet, it doesn't go well since it's a reference effectively outside it's scope so it looks like to do this "right" it's basically an intersphinx reference which might be getting a little ridiculous at this point.

I'm going to go ahead and back that out since it's correct markdown and does the right thing but Sphinx is being overly helpful and causing a different set of problems on the linking side.  Which also explains why the Software Setup stuff doesn't show in the left side bar on the main page since /syseng/ is considered not part of the same docs structure as the top level

Reply:
👍 Thank you for figuring this out and now I know as well :D